common/src/main/java/io/netty/util/concurrent/DefaultThreadFactory.java

```diff
@@ -119,8 +125,9 @@ public Thread newThread(Runnable r) {
         return t;
     }
 
+    // TODO: Once we can break the API we should add ThreadGroup to the arguments of this method.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)